python/mlx/nn/layers/activations.py

```diff
@@ -85,11 +134,41 @@ class ReLU(Module):
     pass
 
 
+@_make_activation_module(leaky_relu)
+class LeakyReLU(Module):
+    pass
```

Comment:
Should be able to pass the alpha in the constructor and subsequently pass it to the leaky_relu function call

Reply:
changed, thanks for the suggestion!